src/java.security.jgss/share/native/libj2gss/GSSLibStub.c

```diff
@@ -210,15 +213,10 @@ gss_channel_bindings_t newGSSCB(JNIEnv *env, jobject jcb) {
 
   if ((*env)->IsInstanceOf(env, jcb, tlsCBCl)) {
       // TLS Channel Binding requires unspecified addrtype=0
-      cb->initiator_addrtype = GSS_C_AF_UNSPEC;
```

Comment:
Thanks for the update. I suggest not to remove this assignment as it helps understand the flow. Or if you prefer it then invert if condition to cover else part and avoid empty if block.